tf-modules/credstash-setup/variables.tf

```diff
@@ -27,3 +31,14 @@ variable "db_table_name" {
   default = "credential-store"
   description = "Name of the DynamoDB table where credentials will be stored"
 }
+
+variable "create_reader_policy" {
+  default = false
+  description = "Should credstash Secret Reader IAM Policy be created."
+}
+
+variable "create_writer_policy" {
+  default = false
+  description = "Should credstash Secret Writer IAM Policy be created."
+}
+
```

Comment:
Q: does it make any sense for a user to create their own reader/write policy?

Q: Would any of this code need to change if we needed to give an EC2 instance a credstash role, _AND_ another role to do something else on AWS (say, access an EBS volume, S3 bucket, etc) ?